src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/Utils/XBindExpressionParser.cs

```diff
@@ -259,6 +271,31 @@ internal static (bool result, CastExpressionSyntax? expression) IsInsideCast(Syn
 
 				return (false, null);
 			}
+
+			internal static (bool result, ParenthesizedExpressionSyntax? expression) IsPathLessCast(SyntaxNode node)
+			{
+				var currentNode = node.Parent;
+
+				do
+				{
+					if (currentNode is ArgumentSyntax arg
+						&& arg.Expression is ParenthesizedExpressionSyntax expressionSyntax)
+					{
+						return (true, expressionSyntax);
+					}
```

Comment:
Consider validating the multiple redundant parentheses case. ie, if `expressionSyntax.Expression` is another `ParenthesizedExpressionSyntax` (etc..), should we unwrap them all and return a non-ParenthesizedExpressionSyntax ?

Reply:
It cannot happen in that context, the user will not provide advanced constructs in that context (they'll fail later on a compile time).